src/components/Book.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBookFetch } from '../redux/books/books';
+import classes from './Book.module.css';
+
+const Book = (props) => {
+  const { id, book } = props;
+  const { title, author, category } = book;
+  const dispatch = useDispatch();
+
+  const removeBook = () => {
+    dispatch(removeBookFetch(id));
+  };
+
+  return (
+    <li className={classes.books}>
+      <div className={classes.book__list}>
+        <h3 className={classes.book__categories}>{category}</h3>
+        <h2>{title}</h2>
+        <h3 className={classes.book__author}>{author}</h3>
+      </div>
+      <button type="button" className={classes.comment__btns}>
+        Comments
+      </button>
+      <button
+        type="button"
+        onClick={removeBook}
+        className={classes.remove__btns}
+      >
+        Remove
+      </button>
+      <button type="button" className={classes.edit__btns}>
+        Edit
+      </button>
+    </li>
```

Comment:
- [ ] There are some components that make up the Book component UI that you have not included even if they should be hard coded. Check out the comparison below 

Given design
![image](https://user-images.githubusercontent.com/90222110/225729403-b6652568-3c2e-4594-81e2-5f6f4cd51ec3.png)

Your design
![image](https://user-images.githubusercontent.com/90222110/225729537-25ce7849-ca38-4445-b9b7-1fa40b522c51.png)